apis/validator/sync_committee_selections.yaml

```diff
@@ -0,0 +1,41 @@
+post:
+  operationId: "submitSyncCommitteeSelections"
+  summary: Determine if a distributed validator has been selected to make a sync committee contribution
+  description: |
+    Submit sync committee selections to a DVT middleware client. It returns the threshold aggregated sync 
+    committee selection. This endpoint should be used by a validator client running as part of a distributed 
+    validator cluster, and is implemented by a distributed validator middleware client. This endpoint is 
+    used to exchange partial selection proof slot signatures for combined/aggregated selection proofs to 
+    allow a validator client to correctly determine if one of its validators has been selected to perform 
+    a sync committee contribution (sync aggregation) duty in this slot. Consensus clients need not support 
+    this endpoint and may return a 501.
+  tags:
+    - Validator
+  requestBody:
+    content:
+      application/json:
+        schema:
+          title: SyncCommitteeSelectionRequest
+          type: array
+          items:
+            $ref: '../../beacon-node-oapi.yaml#/components/schemas/SyncCommitteeSelection'
+  responses:
+    "200":
+      description: |
+        Returns the threshold aggregated sync committee selection proofs.
+      content:
+        application/json:
+          schema:
+            title: SyncCommitteeSelectionResponse
+            type: object
+            properties:
+              data:
+                type: array
+                items:
+                  $ref: '../../beacon-node-oapi.yaml#/components/schemas/SyncCommitteeSelection'
+    "400":
+      $ref: '../../beacon-node-oapi.yaml#/components/responses/InvalidRequest'
+    "500":
+      $ref: '../../beacon-node-oapi.yaml#/components/responses/InternalError'
+    "503":
+      $ref: '../../beacon-node-oapi.yaml#/components/responses/CurrentlySyncing'
```

Comment:
could consider adding 501 error response here, as it is case for `submitBeaconCommitteeSelections`, I guess it was just missed?